taskcluster/scripts/build_qt/compile_qt_6.ps1

```diff
@@ -0,0 +1,89 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+$REPO_ROOT_PATH =resolve-path "$PSScriptRoot/../../../"
+$FETCHES_PATH =resolve-path "$REPO_ROOT_PATH/../../fetches"
+
+
+$BIN_PATH = "$REPO_ROOT_PATH/bin"
+
+Set-Location $FETCHES_PATH 
+Invoke-WebRequest -Uri https://download.qt.io/archive/qt/6.2/6.2.3/single/qt-everywhere-src-6.2.3.zip -OutFile qt-everywhere-src-6.2.3.zip
```

Comment:
can you set a variable to pick the right version? we are migrating to 6.2.4 everywhere.

Reply:
This is not done yet.